python_modules/dagster/dagster/core/instance/config.py

```diff
@@ -127,4 +130,8 @@ def dagster_instance_config_schema():
                 "cancellation_thread_poll_interval_seconds": Field(int, is_required=False),
             },
         ),
+        "code_servers": Field(
+            {"local_startup_timeout": Field(int, is_required=False)},
+            is_required=False,
+        ),
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":100,"endLine":136,"prLink":"https://app.graphite.dev/github/pr/dagster-io/dagster/7387"}} -->
<i class='graphite__hidden'>[Re: lines 100 to 136]<br/></i>

could move this default_value in to the Field definition it self so that its always present <span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/7387?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
I don't think default_values on these fields are behaving the way they are supposed to, I'll look into that separately